app/lib/max_allowed_days_loader.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module MaxAllowedDaysLoader
```

Comment:
`MaxAllowedDaysLoader` name sound a bit not related..

it's like max allowed days loader to what? and why loader?

i would prefer something like: `ContractSuspendConfiguration` or similar.

Reply:
`MaxAllowedDays` works for me.

Reply:
@hallelujah @macejmic I see you are both right that it is not a proper name. Although `ContractSuspendConfiguration` sounds like it is a feature related to the contract and it isn't 😄 
What about `AutomaticallySuspendAndDeleteAccountConfig` ?

Reply:
For the record, later we could use it also for:
https://github.com/3scale/porta/blob/53d21a83f6f341111f4cd3cffac514847ef00a02/config/initializers/audited_hacks.rb#L7
and
https://github.com/3scale/porta/blob/53d21a83f6f341111f4cd3cffac514847ef00a02/app/models/log_entry.rb#L8

Reply:
`AccountSuspensionConfig` ? :)

Reply:
If it is unrelated, it wouldn't use the same file and the same config and the same validity check... it is overcomplicating it and against the single responsibility principle.

Reply:
I called it `AccountSuspensionConfig` at first but I changed it for `AccountDeletionConfig` in the last commit because I feel it suits better, but it is the only thing the last commit does so if you guys disagree, I can remove it easily 😄